### PR TITLE
feat(ld-sidenav): add header and toggle outside to sidenav

### PR DESCRIPTION
**Pull request by borisdiakur**
# Description

This PR moves the toggle which is integrated in the sidenav out of the sidenav component and instead places it in two new components: the sidenav-header and the sidenav-toggle-outside components. 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

- [x] unit tests
- [x] e2e tests
- [x] accessibility tests

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing tests pass locally with my changes

### CODE REVIEW

File: screenshot/builds/master.json [hunk 1]
```diff
@@ -6324,7 +6324,7 @@
     },
     {
       "id": "2fab70cb",
-      "image": "32f760a1fc92bad50def121a0e2de5c4.png",
+      "image": "91263267d15c9b53286d9da2721498c7.png",
```
[the remark this answers is not in the document]
Comment (borisdiakur) [reply screenshot/builds/master.json:6327]: That's intended. The Merck font should not be used here according to branding guidelines.

File: src/liquid/components/ld-notice/assets/ld-input-message-success.svg [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<svg width="24" height="25" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
+<svg width="24" height="25" viewBox="0 0 24 25" fill="none">
```
[the remark this answers is not in the document]
Comment (borisdiakur) [reply src/liquid/components/ld-notice/assets/ld-input-message-success.svg:1]: If it's only three files then it is probably the custom prop. But honestly, it is not important that GitHub can not render a couple of SVGs.

File: src/liquid/components/ld-sidenav/ld-sidenav.tsx [hunk 3]
```diff
@@ -402,9 +443,19 @@ export class LdSidenav {
...
+    setTimeout(() => {
+      this.transitions = true
+    }, 100)
+
+    // When there is less space available than before the breakpoint change,
+    // collapse the side nav if it is collapsible.
+    if (this.closable && this.collapsible) this.collapsed = true
+
     this.ldSidenavBreakpointChange.emit(this.closable)
   }
```
[the remark this answers is not in the document]
Comment (borisdiakur) [reply src/liquid/components/ld-sidenav/ld-sidenav.tsx:458]: What for? I'd rather not make the component more complicated than it is already.

File: src/liquid/components/ld-sidenav/ld-sidenav-navitem/ld-sidenav-navitem.tsx [hunk 4]
```diff
@@ -50,7 +50,7 @@ export class LdSidenavNavitem implements InnerFocusable {
   @Prop() mode?: 'secondary' | 'tertiary'
 
   /** Applies full border-radius. */
-  @Prop() rounded = false
+  @Prop({ reflect: true }) rounded = false
```
Comment (renet) [inline src/liquid/components/ld-sidenav/ld-sidenav-navitem/ld-sidenav-navitem.tsx:53]: What's the advantage of reflecting prop changes here?

**Review by renet — APPROVED** [review]

File: src/liquid/components/ld-sidenav/ld-sidenav-navitem/ld-sidenav-navitem.tsx (hunk 4 above)
Comment (borisdiakur) [reply src/liquid/components/ld-sidenav/ld-sidenav-navitem/ld-sidenav-navitem.tsx:53]: I'm using the attribute as a selector in ld-sidenav-accordion.

Comment (borisdiakur) [reply src/liquid/components/ld-sidenav/ld-sidenav-navitem/ld-sidenav-navitem.tsx:53]: I'm using the attribute as a selector in ld-sidenav-accordion.

**Pull request merged**